### ISSUE TITLE
NPM Security Warning

### ISSUE THREAD

**Issue by Richardsonke** [pepopowitz/your-first-react-app-exercises#2]
When running npm install, it showed a vulnerability warning.  Running `npm audit` returns the following:

```
                       === npm audit security report ===

# Run  npm install webpack-dev-server@3.1.14  to resolve 1 vulnerability

  High            Missing Origin Validation

  Package         webpack-dev-server

  Dependency of   webpack-dev-server

  Path            webpack-dev-server

  More info       https://nodesecurity.io/advisories/725
```

Can we/should we upgrade webpack-dev-server?

**Comment by petdance** [comment]
I got this, too.  

```
$ npm install

> fsevents@1.2.4 install /Users/andy/your-first-react-app-exercises/node_modules/fsevents
> node install

[fsevents] Success: "/Users/andy/your-first-react-app-exercises/node_modules/fsevents/lib/binding/Release/node-v64-darwin-x64/fse.node" already installed
Pass --update-binary to reinstall or --build-from-source to recompile

> styled-components@4.1.3 postinstall /Users/andy/your-first-react-app-exercises/node_modules/styled-components
> node ./scripts/postinstall.js || exit 0

Use styled-components at work? Consider supporting our development efforts at https://opencollective.com/styled-components
added 1844 packages from 797 contributors and audited 36301 packages in 45.556s
found 1 high severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
```

And then running `npm audit` got the results shown above by @Richardsonke

**Comment by pepopowitz** [comment]
Thanks for the heads-up, friends! 

This should be resolved with a522410349957d14609d36784cfd3c38961f3a40.
